plugins/python-build/bin/python-build

```diff
@@ -934,7 +934,7 @@ build_package_micropython() {
   { cd mpy-cross
     "$MAKE" $MAKE_OPTS
     cd ../ports/unix
-    "$MAKE" $MAKE_OPTS axtls
+    grep -qE '^axtls:' Makefile && "$MAKE" $MAKE_OPTS axtls
```

Comment:
A comment explaining the idea behind this would be welcome.
From what I saw, this was needed in some old versions.

Reply:
In the latest few versions, installing MicroPython doesn't require running the `make axtls` command; otherwise, it might result in an error. This change checks the makefile for dependencies on axtls to determine if the `make axtls` command needs to be executed.

Reply:
> In the latest few versions, installing MicroPython doesn't require running the `make axtls` command; otherwise, it might result in an error. This change checks the makefile for dependencies on axtls to determine if the `make axtls` command needs to be executed.

Yes. I meant, a comment in the code saying that briefly would be welcome -- 'cuz that knowledge is a specter (piece of knowledge crucial for the code to work that cannot be expressed in the code itself).